samples/star/build.gradle.kts

```diff
@@ -115,7 +115,7 @@ dependencies {
   testImplementation(libs.androidx.compose.ui.testing.manifest)
   testImplementation(libs.leakcanary.android.instrumentation)
 
-  debugImplementation(libs.androidx.compose.ui.testing.manifest)
+  androidTestImplementation(libs.androidx.compose.ui.testing.manifest)
```

Comment:
Without this, `ComposeActivity` isn't merged with the manifest meaning it can't be found at runtime. Before introduction of the apk module, we used to support debug builds.

Reply:
I think it needs to be present in both?

Reply:
By both do you mean unit tests? We were already merging [correctly](https://github.com/slackhq/circuit/pull/275/files/9978fe2c84c0299c9ff14359acfe4bde665dee91#diff-755de7e3dcd69a6fb7e82f9ba3679f9cdcf927dda2dc05d0a6bd61049257639dR115) for unit tests